microsoft-edge/visual-studio-code/elements-for-edge.md

```diff
@@ -0,0 +1,69 @@
+---
+description: How to use Elements for Microsoft Edge (Chromium) from VS Code
+title: Elements for Microsoft Edge (Chromium) from VS Code
+author: erdraud
+ms.author: erdraud
+ms.date: 08/08/2019
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, elements
+---
+
+# Elements for Edge VS Code extension
+
+By adding the [Elements for Edge](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools) VS Code extension, you can use the browser's Elements tool from within Visual Studio Code. By either launching or attaching, the Elements tool will connect to an instance of Microsoft Edge, display the runtime HTML structure, and allow you to alter the layout or fix styling issues.
+
+![GIF of the Elements for Edge VS Code extension at work](./media/elements-for-edge.gif)
+
+## Launching Edge From the Elements tab 
```

Comment:
We should probably be consistent between Elements tool and Elements tab and Debugger tool and Debugger tab in the titles.